content/docs/csidriver/release/powerstore.md

```diff
@@ -28,6 +28,7 @@ description: Release notes for PowerStore CSI driver
 | fsGroupPolicy may not work as expected without root privileges for NFS only<br/>https://github.com/kubernetes/examples/issues/260          | To get the desired behavior set "allowRoot: "true" in the storage class parameter                                                                                                                                                                                                                                                       |
 | If the NVMeFC pod is not getting created and the host looses the ssh connection, causing the driver pods to go to error state              | remove the nvme_tcp module from the host incase of NVMeFC connection                                                                                                                                                                                                                                                                    |
 | When a node goes down, the block volumes attached to the node cannot be attached to another node                                           | This is a known issue and has been reported at https://github.com/kubernetes-csi/external-attacher/issues/215. Workaround: <br /> 1. Force delete the pod running on the node that went down <br /> 2. Delete the volumeattachment to the node that went down. <br /> Now the volume can be attached to the new node.                   |
+| When driver node pods enter CrashLoopBackOff and PVC remains in pending state with one of the following events:<br /> 1. failed to provision volume with StorageClass `<storage-class-name>`: error generating accessibility requirements: no available topology found <br /> 2. waiting for a volume to be created, either by external provisioner "csi-powerstore.dellemc.com" or manually created by system administrator.  | Check whether all array details present in the secret file are valid and  remove any invalid entries if present. Redeploy the driver. This issue has been reported at https://github.com/dell/csm/issues/871                                                                                                                                                                                                                                                             
```

Comment:
Extra space between "are valid and       remove any invalid..."  

Please change #871 to be a feature/enhancement and remove "This issue has been reported at..." and remove "This issue has been reported at https://github.com/dell/csm/issues/871"

Reply:
done